sycl/include/sycl/marray.hpp

```diff
@@ -132,7 +132,7 @@ template <typename Type, std::size_t NumElements> class marray {
 
   // Available only when: NumElements == 1
   template <std::size_t Size = NumElements,
-            typename = typename std::enable_if<Size == 1>>
+            typename = typename std::enable_if_t<Size == 1>>
```

Comment:
Is the rhs `typename` still required here?

Reply:
Good eye! No, it should be fine without. It has been removed!